src/main/java/me/gallowsdove/foxymachines/implementation/weapons/Elucidator.java

```diff
@@ -0,0 +1,48 @@
+package me.gallowsdove.foxymachines.implementation.weapons;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import me.gallowsdove.foxymachines.Items;
+import me.gallowsdove.foxymachines.utils.Utils;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Elucidator extends OnHitWeapon {
+    public Elucidator(boolean customMobs) {
```

Comment:
Please change this parameter name to something that implies it's a boolean, not a set of mobs

Reply:
Can do